app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileViewModel.kt

```diff
@@ -338,6 +341,10 @@ class SelfUserProfileViewModel @Inject constructor(
         userProfileState = userProfileState.copy(errorMessageCode = null)
     }
 
+    fun trackQrCodeClick() {
+        analyticsManager.sendEvent(AnalyticsEvent.QrCode.Click(!userProfileState.teamName.isNullOrBlank()))
```

Comment:
can I assume its a personal account when team name is null?

Reply:
it is better to look at team id, also every team have a name i am not 100% sure if this is enforced by the backend and getting the team id is a safe option
@m-zagorski

Reply:
Thanks, currently `SelfUserProfile` have:
```
val teamName: String? = "", // maybe teamId is better here
```
I believe we should do that change then- but most likely in a separate PR